modules/op-specifying-pipelines-resource-quota-using-priority-class.adoc

```diff
@@ -185,7 +185,7 @@ spec:
 .Example: Verify resource quota usage for the pipeline
 [source,terminal]
 ----
-$ kubectl describe quota
+$ oc describe quota
```

Comment:
From the OCP doc standards "Separate a command and its related example output into individual code blocks."